js/Domupdates.js

```diff
@@ -155,40 +156,31 @@ let domUpdates = {
     // insert letter into each box as inner text
     // give CSS property of white background and white text
     // if they guess the correct letter, letter changes to black text
-    let splitAnswerArray = game.getCurrentPuzzle().currentPuzzle.correct_answer.toUpperCase().split('');
-    let greenPuzzleBoxes = document.getElementsByClassName('puzzle-box');
-    for(let i = 0; i < splitAnswerArray.length; i++) {
-      if (splitAnswerArray[i] === ' ') {
-        greenPuzzleBoxes[i].classList.add('green-background')
-      }
-      if (splitAnswerArray[i] === '-') {
-        greenPuzzleBoxes[i].innerText = '-'
-      }
-      if (splitAnswerArray[i] === '&') {
-        greenPuzzleBoxes[i].innerText = '&'
-      }
-      greenPuzzleBoxes[i].classList.add('white-background');
-    }
-    splitAnswerArray.forEach((letter, index) => {
-      if (index  < 12) {
-        let rowBox = document.querySelector(`.row1-box${index + 1}`);  
-        rowBox.innerText = letter
-      } else if (index < 26) {
-        let rowBox = document.querySelector(`.row2-box${index - 11}`);
-        rowBox.innerText = letter
-      }
-  })
-},
+    
+    console.log(game.getCurrentPuzzle().currentPuzzle.correct_answer)
 
-  removePuzzle() {
     let splitAnswerArray = game.getCurrentPuzzle().currentPuzzle.correct_answer.toUpperCase().split('');
-    let greenPuzzleBoxes = document.getElementsByClassName('puzzle-box');
-    for (let i = 0; i < splitAnswerArray.length; i++) {
-      greenPuzzleBoxes.classList.add('white-background')
-      greenPuzzleBoxes.innerText = ''
-    }
-  },
-
+    splitAnswerArray.forEach((letter, index) => {
+      if (letter === ' ') {
+        this.greenPuzzleBoxes[index].classList.remove('white-background')
+        this.greenPuzzleBoxes[index].classList.add('green-background')
+      } else if (letter === '-' || letter === '&' || letter === '\'') {
```

Comment:
Ah nice refactor here. Much easier to understand. No need to change the specifically change the inner text to those characters since we are treating them as letters anyway